Add SLO for PostgreSQL by VSHN (#142)
vshn/component-appcat: docs/modules/ROOT/pages/references/slo-parameters.adoc

The input passed to the Kustomize renderer.
See https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/[the Kustomization File] for all available options.

== `alerting.labels`
[horizontal]
type:: dict

Labels that will be added to any SLO alerts.

== `alerting.page_labels`
[horizontal]
type:: dict

Labels that will be added to any critical SLO alerts.
`page_alerts` are alerts are critical alerts for a high burn rate that require immediate attention.

== `alerting.tickt_labels`
[horizontal]
type:: dict

Labels that will be added to any non-crtiical SLO alerts.

> `ticket_alerts` are alerts are alerts for an elevated burn rate that might require attention, but aren’t urg

Reviewer: ```suggestion
`ticket_alerts` are alerts for an elevated burn rate that might require attention, but aren’t urgent.
```

== `vshn.postgres.uptime`

Configuration of the PostgreSQL by VSHN uptime SLO.

=== `vshn.postgres.uptime.objective`
[horizontal]
type:: float
default:: 99.9

The SLO objective in percentage between 0-100.

=== `vshn.postgres.uptime.alerting.page_alert`
[horizontal]
type:: dict

The alert configuration for `page_alerts`.

> Can be used add additional lables or annotations throght the `labels` or `annotations` key, disable the alert through the `disable` key, or delay emitting the alert throug the `for` key.

Reviewer: ```suggestion
Can be used to add additional labels or annotations through `labels` or `annotations` key, disable the alert through the `disable` key, or delay emitting the alert through the `for` key.
```
This is a bit confusing, I corrected the typos but still not very clear. Can you rephrase that?

=== `vshn.postgres.uptime.alerting.ticket_alert`
[horizontal]
type:: dict

> The alert configuration for `page_alerts`.

Reviewer: ```suggestion
The alert configuration for `ticket_alert`.
```

> Can be used add additional lables or annotations throght the `labels` or `annotations` key, disable the alert through the `disable` key, or delay emitting the alert throug the `for` key.

Reviewer: Again the same typos.
